src/databricks/labs/ucx/source_code/notebooks/sources.py

```diff
@@ -111,7 +125,29 @@ def lint(self) -> Iterable[Advice]:
     def _linter(self, language: Language) -> Linter:
         if language is Language.PYTHON:
             return self._python_linter
-        return self._languages.linter(language)
+        return self._context.linter(language)
+
+    def _load_source_from_run_cell(self, run_cell: RunCell):
+        path, _, _ = run_cell.read_notebook_path()
+        if path is None:
+            return  # malformed run cell already reported
+        resolved = self._path_lookup.resolve(path)
+        if resolved is None:
+            return  # already reported during dependency building
+        # TODO deal with workspace notebooks
```

Comment:
`resolved` already has workspace notebook because of WorkspacePath.

Reply:
noted.